modules/ROOT/pages/policies-policy-categories.adoc

```diff
@@ -0,0 +1,39 @@
+= Policy Categories
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are categorized by the funtion they perform. For example, becuase you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a Troubleshooting policy. The following table lists the Default policy name, its category and the function that it performs. +
+You can apply only one policy of each category at a given time.
+
+[%header,cols="40a,15a,60a"]
+|===
+| Policy | Category | Function
+| Basic Authentication - LDAP | Security | Authenticates the LDAP credentials.
+| Basic Authentication - Simple | Security | Authenticates single user password.
+| Client ID Enforcement | Compliance | Allows access to client applications with a valid Client ID.
```

Comment:
I would say "with valid client credentials" because you can validate client id AND client secret, or only the client id if the policy is configured with secret as optional

Reply:
Done! Thanks!